styleguide/derek/incubation/homepage.ejs

```diff
@@ -5,6 +5,38 @@
   'body': 'Redhat names Rackspace North America Managed Service Partner of the Year'
 }) %>
 
+<!-- Case Study -->
+
+<div class="caseStudy-wrapper">
+    <div class="container">
+        <div class="caseStudy-main-image"></div>
+        <div class="row-eq-height">
+          <div class="caseStudy-box caseStudy-header col-md-3 col-sm-6">Helping our customers get to great</div>
+          <div class="caseStudy-box col-md-3 col-md-offset-6 col-sm-6">
+            <img class="caseStudy-image" src="/imgs/derek/logos/feeding-america-blk.png"/>
+            <p class="caseStudy-text">Feeding America migrated over a petabyte of its data to a serverless cloud architecture, eliminating hardware maintenance and reducing capital costs.</p>
+            <a class="caseStudy-button">Read their story &gt;</a>
+          </div>
+        </div>
+        <div class="row-eq-height">
+          <div class="caseStudy-box col-md-3 col-md-offset-3 col-sm-6">
+            <div class="caseStudy-bar">
+              <img class="caseStudy-image" src="/imgs/derek/logos/mckesson-blk.png"/>
+              <p class="caseStudy-text">McKesson simplified its dated, complex workflow by migrating data to the cloud, and transformed its management abilities.</p>
+              <a class="caseStudy-button">Read their story &gt;</a>
+            </div>
+          </div>
+          <div class="caseStudy-box col-md-3 col-md-offset-3 col-sm-6">
+            <div class="caseStudy-bar">
+            <img class="caseStudy-image" src="/imgs/derek/logos/aeromexico-blk.png"/>
```

Comment:
Make sure we delete any unused images from the repo.  For example, you're only using `aeromexico-blk.png` but you've included `aeromexico.png` in the repo, despite not using it.